option optimize_for = LITE_RUNTIME;

message BlusherOperRequest
{
 required uint32 id = 1;
}
